src/lib.rs

```diff
@@ -1323,6 +1318,22 @@ impl yaxpeax_arch::Instruction for InstructionBundle {
             insn.predicate() == 0
         };
 
+        for insn in &self.instructions {
+            match insn.opcode {
+                // B8 family instructions cannot be predicated
+                Opcode::Cover | Opcode::Clrrb | Opcode::Clrrb_pr | Opcode::Rfi | Opcode::Bsw_0 | Opcode::Bsw_1 | Opcode::Epc => {
+                    if insn.predicate() != 0 {
+                        return false;
+                    }
+                },
+                // Undefined opcode regions
```

Comment:
ah yes, these definitely should not be `well_formed`, thank you for adding them here.